kaspresso/src/main/kotlin/com/kaspersky/kaspresso/runner/listener/KaspressoLateRunListener.kt

```diff
@@ -0,0 +1,31 @@
+package com.kaspersky.kaspresso.runner.listener
+
+import org.junit.runner.Description
+
+interface KaspressoLateRunListener : KaspressoRunListener {
+
+    fun lateInit(cache: Cache) {
+        testRunStarted(cache.testRunStartedDescription)
+        testStarted(cache.firstTestStartedDescription)
+    }
+
+    class Cache {
```

Comment:
Add some info in comments what is it about. Move to notifier class

Reply:
Done